src/Client.php

```diff
@@ -175,6 +175,20 @@ public static function instance($name = 'default')
         return static::$instances[$name];
     }
 
+    /**
+     * @param string $name
+     *
+     * @return bool true if an instance has been found and removed
+     */
+    public static function deleteInstance($name)
```

Comment:
What if this deleted the `default` instance by default?

```php
public static function deleteInstance($name = 'default')
```

That'd make it more convenient for most implementations.

Reply:
True, will add it in :+1: